python/stempy/io/__init__.py

```diff
@@ -37,6 +37,14 @@ def read(self):
 def reader(path, version=FileVersion.VERSION1):
     return Reader(path, version)
 
+def expand_electron_data(frames, numPixels):
+    ret = np.zeros((len(frames) * numPixels,), dtype=np.uint16)
+    for i, frame in enumerate(frames):
+        for pos in frame:
+            ret[i * numPixels + pos] = 1
+
+    return ret
+
```

Comment:
Shouldn't we do this expansion in C++, given the looping?

Reply:
We can move it to C++, although I don't think this function takes up too much time. ~0.19 seconds in my timing (the STEM calculation is taking around 50 seconds).

Reply:
ok, that is fine, leave for the moment

Reply:
50 seconds on the electron counted data?